### ISSUE TITLE
Cannot log in to the Backend with admin account (OM 20.0 only)

### ISSUE THREAD

**Issue by ADDISON74** [OpenMage/magento-lts#2313]
**Steps to reproduce the issue**

1. Create two directories. One for OpenMage 20.14 released as archive version on May 18th and one for branch 20.0 from today (cloned or archive).
2. Install both versions from scratch and try to log in to the Backend using the admin account.

**Results**
OpenMage 20 released in May allows access while today's version of OpenMage does not and reloads the empty form again in a loop. I tried to reset the password without any positive results.

I repeated the test with today's version of OM 1.9.4.16 and I was able to access the Backend. Most likely what has been added so far to OM 20 after May 18t has created this issue. It is quite difficult to identify quickly because no errors are reported in the log files, but I will eventually find the source.

It will be great if anyone else can do a similar test to confirm the issue. If it is confirmed no one should use the latest up-to-date version of the repository in production, but stay with the May version.

**Comment by fballiano** [comment]
I logged-in on a 20.0 branch checkout right now:

<img width="1171" alt="Schermata 2022-07-13 alle 11 20 51" src="https://user-images.githubusercontent.com/909743/178698938-0d1ed58d-f71f-4c67-825e-9beb24949650.png">

**Comment by ADDISON74** [comment]
I could do it until yesterday when I brought the latest version locally. When I saw that I couldn't connect, I didn't stop to analyze and I checked if I could do it in another installed version. With May version everything is fine. I deleted and cloned the repository. After installation I had the same problem. 

If you do not confirm more then I close the post and I will open it again after I identify the files that generated the issue.
